packages/backend/src/clients/EmailClient/EmailClient.ts

```diff
@@ -82,7 +82,7 @@ export default class EmailClient {
                 hbs({
                     viewEngine: {
                         partialsDir: path.join(__dirname, './templates/'),
-                        defaultLayout: false,
+                        defaultLayout: undefined,
```

Comment:
is this change relevant ?

Reply:
there was a TypeScript error.. Once I set up pnpm, type errors started to pop. I guess that's because of the flat-structured node_modules; it wasn't referring to the correct package version I guess